xbmc/music/dialogs/GUIDialogSongInfo.cpp

```diff
@@ -58,10 +58,13 @@ CGUIDialogSongInfo::CGUIDialogSongInfo(void)
   m_needsUpdate = false;
   m_startUserrating = -1;
   m_loadType = KEEP_IN_MEMORY;
+  m_songContributors = new CFileItemList;
```

Comment:
Because you don't have to care about calling delete, it's taken care of by the smart pointer.